client/stack-editor/lib/index.coffee

```diff
@@ -136,10 +136,13 @@ module.exports = class StackEditorAppController extends AppController
     editor?.destroy()
 
 
-  reloadEditor: (template) ->
+  reloadEditor: (templateId) ->
 
-    @removeEditor template._id
-    @showView template
+    { computeController } = kd.singletons
+
+    computeController.fetchStackTemplate templateId, (err, template) =>
```

Comment:
Please add a action wrapping this function, and then call that action here. The result of this call will not update the stores.